packages/Operators/test/tstConsistentInterpolationOperator.cpp

```diff
@@ -65,8 +68,125 @@
 #include <Tpetra_Map.hpp>
 
 //---------------------------------------------------------------------------//
+// TEST EPSILON
+//---------------------------------------------------------------------------//
+
+const double epsilon = 1.0e-14;
+
+//---------------------------------------------------------------------------//
 // Tests
 //---------------------------------------------------------------------------//
+TEUCHOS_UNIT_TEST( ConsistentInterpolationOperator, reference_hex_mesh )
```

Comment:
c.f. comments I did for the `L2ProjectionOperator` regarding the declaration of a function that evaluates the field and encapsulates `data_x, data_y, etc.` and the construction of the axis aligned `ReferenceHexMesh` from two opposite corner points and the numbers of subdivisions in each direction.